plugins/rbac/src/components/RolesList/RolesList.tsx

```diff
@@ -47,6 +48,11 @@ export const RolesList = () => {
   return (
     <>
       <SnackbarAlert toastMessage={toastMessage} onAlertClose={onAlertClose} />
+      {error?.roleConditionError && (
+        <Box>
+          <Alert severity="warning">{error?.roleConditionError}</Alert>
+        </Box>
+      )}
```

Comment:
@ciiay there is already a warning panel below RolesListToolbar for simple policies error why dont you use that instead of adding a new alert?

Reply:
Updated screenshot
![image](https://github.com/janus-idp/backstage-plugins/assets/26255262/f696fb1e-a7bb-4d06-a64f-9d222325924d)